src/Error.js

```diff
@@ -0,0 +1,39 @@
+class ErrorCase {
+  caseTypes = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+  convertErrorMessage = '/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>';
```

Comment:
The `convertErrorMessage` string uses a placeholder format that might not be correctly interpreted if used directly in a URL. Consider using a more explicit format or providing an example of how the URL should be structured.